### ISSUE TITLE
Action Bar, Action Group and Action - Provide a valid label for form fields - (2036569764)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5555]
### Module: 
 05 Action Bar, Action Group and Action 

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036569764 

### Severity: 
 10 

### Description: 
 [Issue]
There are buttons with identical accessible names. Examples include:

- Information action, for display purposes only
- Feedback action, for display purposes only
- Expand

[User Impact]
Screen reader users will be unable to determine the purpose of these buttons. Speech input users will have difficulty activating them.

[Code Reference] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Information action, for display purposes only" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Information</div>
</button>
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with internal text, an aria-label attribute. Good accessible names are both concise and descriptive.

[Compliant Code Example] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Information action bar, for display purposes only" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Information</div>
</button>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
> There are buttons with identical accessible names. 

This seems like just an end user problem and not a component issue.

Is this issue valid?

**Comment by geospatialem** [comment]
> This seems like just an end user problem and not a component issue.
> 
> Is this issue valid?

Good catch, @driskull. We're loading in all the issues from the audit, and once loaded in will verify individual issues to determine next steps and prioritization.

As you mention, this was a result of the component tests for the audit, and no further action is needed. Will close this one. 👍🏻

**Comment by driskull** [comment]
@geospatialem Just to be sure, its not the inner text and the aria-label being the same that is the issue?

```
<button aria-busy="false" aria-disabled="false" aria-label="My Text" class="button">
 <div aria-hidden="true" class="icon-container">
   …
 </div>
 <div class="text-container">My Text</div>
</button>
```

Like that?

Should we be auditing the a11y report before creating issues?

**Comment by geospatialem** [comment]
> @geospatialem Just to be sure, its not the inner text and the aria-label being the same that is the issue?

Think it's duplicate names used throughout the tests. 

@driskull Here's the [code used in the test](https://github.com/geospatialem/calcite-samples/blob/main/action-bar.html) where the same labels were used for each test. Thought it would help with testing for consistent naming for each test, but triggered a few extra issues instead. 😞 

> Should we be auditing the a11y report before creating issues?

Will be working on verification, triage, and prioritization for the issues shortly. Thought it was best to have all the issues populate in an [epic](https://github.com/Esri/calcite-components/issues/4599) so we can track progress as we continue to update our conformance report prior to the 1.0 release.
